src/components/expandable-text/expandable-text.js

```diff
@@ -86,7 +86,7 @@ const ExpandableText = withText({
   }
 
   return (
-    <div>
+    <div className={['expandable-text-container', isTextExpanded ? 'text-expanded' : ''].join(' ')}>
```

Comment:
hey @semarche-kaltura , is it possible to write the class names from styles like we usually do, instead of hard-coding them? or export them to const?